packages/network-of-terms-reconciliation/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@netwerk-digitaal-erfgoed/network-of-terms-reconciliation",
+  "description": "A Reconciliation API for the Network of Terms",
```

Comment:
Or: 'Network of Terms Reconciliation Service API' (similar to the description of the GraphQL package, 'Network of Terms GraphQL API')?